### PR TITLE
Add fields of test pm01

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
## What

* Add few fields in `postmortem_test.go`

* The keys(e.g. `noteworthycustomerimpacted`...) are just my opinion. Feel free to request changes.

## Why

* Test coverage

### CODE REVIEW

File: postmortem/postmortem_test.go [hunk 1]
```diff
@@ -38,6 +38,16 @@ func TestPM01(t *testing.T) {
...
+		{"timetoresponse", "15"},
+		{"timetomigrate", "60"},
+		{"slolink", "http://example.com/slodashboard"},
+		{"impactedproducts", "PizzaDelivery"},
+		{"firstknownimpactedservice", "PizzaBE"},
+		{"knownservicesblastradius", "PizzaFE"},
+		{"noteworthycustomerimpacted", "N/A"},
+		{"trigger", "Deploy"},
+
 		// TODO(stratus): Include all fields.
```
Comment (stratus) [inline postmortem/postmortem_test.go:51]: Should we remove the TODO comment?

**Review by stratus — COMMENTED** [review]
JFTR, re: adding go.sum since the description doesn't include it.
https://github.com/golang/go/wiki/Modules#should-i-commit-my-gosum-file-as-well-as-my-gomod-file

"Typically your module's go.sum file should be committed along with your go.mod file."

File: postmortem/postmortem_test.go (hunk 1 above)
Comment (KeisukeYamashita) [reply postmortem/postmortem_test.go:51]: The `Action Item` is still missing so in my opinion, we should leave the `TODO` here

Comment (stratus) [reply postmortem/postmortem_test.go:51]: Makes sense, thanks.

**Pull request merged**